iOS/Flows/Conversation/Threads/ConversationThreadViewController+Updates.swift

```diff
@@ -45,66 +44,4 @@ extension ConversationThreadViewController {
             self.messageInputAccessoryView.updateTypingActivity(with: nonMeUsers)
         }.store(in: &self.cancellables)
     }
-
-    /// Updates the datasource with the passed in array of message changes.
-    func update(with changes: [ListChange<Message>],
```

Comment:
This is no longer needed because the datasource has a common update function for conversations and threads.